composer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "workos/workos-php-laravel",
+    "description": "WorkOS PHP Library for Laravel",
+    "keywords": ["laravel", "laravel 5", "laravel 6", "laravel 7", "workos", "sdk", "sso"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "WorkOS",
+            "email": "eng@workos.com"
```

Comment:
I just changed the Ruby SDK to `support@workos.com`—I think we should start using that for public-facing emails